html-forms/index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>HTML Forms</title>
+</head>
+<body>
+  <h1>Coding Form!</h1>
+  <form action="" method="get"></form>
+  <div>
+    <label for="email">Email</label>
+    <br>
+    <input type="text" name="email">
```

Comment:
make sure that each input has an `id` attribute which matches the `for` attribute on the label to get the label functionality!